### PR TITLE
NTER-1330 Bootstrap version 1

### PR DESCRIPTION
**Pull request by manchuck**
Set up some of the documents for V1 Bootstrapping

### CODE REVIEW

File: data/docs/v1/components/parameters/perPage.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "per_page",
```
Comment (hwalpurgis) [inline data/docs/v1/components/parameters/perPage.json]: Shouldn't this be "page_size".?

**Comment by manchuck** [comment]
@hwalpurgis I updated the parameter to be correct. @dilorc2 you will have to rebase your branch

File: data/docs/v1/components/schemas/product.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    },
+    "TrackingType": {
+      "description": "Tracking type",
+      "type": "string",
+      "example": "Face down exit assembly"
+    },
+    "total_inventory": {
+      "description": "Total Inventory",
+      "type": "number",
+      "format": "int32",
```
Comment (manchuck) [inline data/docs/v1/components/schemas/product.json]: this is read-only

File: data/docs/v1/components/schemas/product.json [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+      "minimum": 0
+    },
+    "total_inventory_status": {
+      "description": "Total Inventory",
```
Comment (manchuck) [inline data/docs/v1/components/schemas/product.json]: this is read-only

**Pull request merged**